app/views/reviews/_review_editor.html.erb

```diff
@@ -172,14 +172,16 @@
           <span data-bind="text: averageGrade"></span>
         </h3>
       <!-- /ko -->
-      
+
       <!-- Next page button -->
-      <!-- ko if: nextPage || $root.finishable() -->
-        <button class="btn" data-bind="click: $root.showNextPage">
+        <button class="btn" data-bind="enable: nextPage || $root.finishable(), click: $root.showNextPage">
```

Comment:
I'm confused about the comparison view. For example these are marked with a +:
+    t.string   "positive_caption"
+    t.string   "negative_caption"
+    t.string   "neutral_caption"

Apparently they are there in master, so are these additions or deletions? They are deprecated columns, nevertheless.